CMSgov/mint-app
# [EASI-3671] Update precedence rules (backend)

# EASI-3671

## Changes and Description

- Updated gql schema
- Created flyway migration adding new enum type and table columns
- Updated data model
- Updated Postman collection



## How to test this change

- Bring servers up
- Verify migration succeeded
- Run plan beneficiaries update endpoint in Postman
- Verify success and data in response

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
